samples-generator/bin/3d-tiles-samples-generator.js

```diff
@@ -196,6 +202,13 @@ var ulTileOptions = {
 };
 
 var promises = [
+    // CESIUM_3dtiles_batch_table
+    createBatchedWithBatchTableGltf(),
+    createBatchedWithBatchTableBinaryGltf(),
```

Comment:
Are the binary properties getting added to the glTF? I just see `id`, `Longitude`, `Latitude`, and `Height`.

Reply:
They are now